geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -155,7 +167,7 @@
                         <phase>package</phase>
                         <configuration>
                             <attach>true</attach>
-                            <finalName>geomesa-accumulo-${project.version}</finalName>
+                            <finalName>geomesa-accumulo_${scala.binary.version}-${project.version}</finalName>
```

Comment:
i would prefer to keep the hardcoding constrained to `project.artifactId` and `parent.project.artifactId` until all the version switching stuff is resolved.  This is in case a spark-like `sed` won't work and we have to use something context dependent like XSLT